core/descriptors/src/org/jetbrains/kotlin/renderer/RenderingUtils.kt

```diff
@@ -28,7 +28,7 @@ private fun Name.shouldBeEscaped(): Boolean {
     if (isSpecial) return false
 
     val string = asString()
-    return string in KeywordStringsGenerated.KEYWORDS || string.any { !Character.isLetterOrDigit(it) && it != '_' }
+    return string in KeywordStringsGenerated.KEYWORDS || string.any { !Character.isLetterOrDigit(it) }
```

Comment:
Look at https://teamcity.jetbrains.com/viewLog.html?buildId=2319666
This is incorect fix.